### PR TITLE
Get PRs from GraphQL API

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by beckermr** [comment]
Also, did you try this locally?

**Comment by beckermr** [comment]
The tests in this repo are not very complete so trying the code locally is def a good idea.

File: conda_forge_tick/make_graph.py [hunk 1]
```diff
@@ -340,8 +389,17 @@ def _update_pr(update_function, dry_run, gx):
                 pr_json = migration.get("PR", None)
                 # allow for false
                 if pr_json:
-                    future = pool.submit(update_function, ghctx, pr_json, gh, dry_run)
-                    futures[future] = (node_id, i)
+                    if '__lazy_json__' in pr_json and pr_json['__lazy_json__'].startswith("pr_json/"):
+                        pr_id = int(pr_json['__lazy_json__'].split("/")[1].split(".")[0])
```
Comment (CJ-Wright) [inline conda_forge_tick/make_graph.py]: Some of these may not be valid, since we spoof PRs with uuid4 ids

File: conda_forge_tick/make_graph.py [hunk 2]
```diff
@@ -324,6 +325,44 @@ def make_graph(names: List[str], gx: Optional[nx.DiGraph] = None) -> nx.DiGraph:
     return gx
 
 
+def _get_last_updated_prs():
+    query = textwrap.dedent("""
+        {
+          user(login: "regro-cf-autotick-bot") {
+            pullRequests(first: 100, states: [CLOSED, MERGED], orderBy: {field: UPDATED_AT, direction: DESC} ) {
```
Comment (CJ-Wright) [inline conda_forge_tick/make_graph.py]: Would it be possible to have a query that gets PRs with the bot-rerun label?

Comment (isuruf) [reply conda_forge_tick/make_graph.py]: Yes, you can filter by label.

File: conda_forge_tick/make_graph.py [hunk 3]
```diff
@@ -340,8 +391,21 @@ def _update_pr(update_function, dry_run, gx):
                 pr_json = migration.get("PR", None)
                 # allow for false
                 if pr_json:
-                    future = pool.submit(update_function, ghctx, pr_json, gh, dry_run)
-                    futures[future] = (node_id, i)
+                    if '__lazy_json__' in pr_json:
+                        m = pr_json_regex.match(pr_json['__lazy_json__']):
```
Comment (CJ-Wright) [inline conda_forge_tick/make_graph.py]:
```suggestion
                        m = pr_json_regex.match(pr_json['__lazy_json__'])
```

**Comment by beckermr** [comment]
Alright, let's merge this and see what gives.

**Pull request merged**